docs/next/util/gtag.js

```diff
@@ -1,4 +1,5 @@
 export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_DOCS_GA_ID;
+export const GTM_ID = process.env.NEXT_PUBLIC_DOCS_GTM_ID;
```

Comment:
This env already exists but it was never actually inserted anywhere
![image](https://github.com/dagster-io/dagster/assets/2286579/edcd1a08-bb19-41c4-8c2a-89dbccf941e4)